UI/Components/ServerComponent.cs

```diff
@@ -387,21 +404,30 @@ private void ProcessMessage(String message, Connection clientConnection)
                             title = options[1];
                         }
 
-                        State.Run[index].Name = title;
-                        State.Run.HasChanged = true;
+                        if (index >= 0)
```

Comment:
I think we might still run into an error when the run is ended, since the current split index gets set to `State.Run.Count` in that scenario.

Reply:
That would be existing behavior then, so this at least would be fixing half of the bug. I'll admit this is the one change I didn't test very well since this command is undocumented and I never use it.

Reply:
Could we check if `index >= 0 && index < State.Run.Count`, to avoid errors?